docs/examples/101/azurefirewall-create-with-zones/main.bicep

```diff
@@ -45,7 +45,7 @@ resource publicIP 'Microsoft.Network/publicIPAddresses@2020-06-01' = {
 resource firewall 'Microsoft.Network/azureFirewalls@2020-06-01' = {
   name: firewallName
   location: location
-  zones: length(availabilityZones) == 0 ? json('null') : availabilityZones
+  zones: length(availabilityZones) == 0 ? any(null) : availabilityZones
```

Comment:
>any [](http://example.com/codeflow?start=42&length=3)

This is technically a breaking change. I'm fine with it, but we should call it out on the next release.